code/modules/recycling/disposal-unit.dm

```diff
@@ -379,6 +379,10 @@
 	else if(full_pressure)
 		add_overlay("dispover-ready")
 
+/obj/machinery/disposal/bin/proc/do_flush()
```

Comment:
unless you mean remove the proc and just reinsert the code into the calling proc